packages/ember-runtime/tests/computed/reduce_computed_macros_test.js

```diff
@@ -427,6 +427,23 @@ QUnit.test('properties values can be replaced', function() {
   deepEqual(obj.get('a1bs').mapBy('name'), ['item1'], 'properties can be filtered by matching value');
 });
 
+QUnit.test('responds to change of property value on element', function() {
+  let a1 = { foo: true };
+  let a2 = { foo: true };
+
+  obj = EmberObject.extend({
+    a: filterBy('array', 'foo')
+  }).create({
+    array: [a1, a2]
```

Comment:
The test fails because we do not use prototype extensions by default in the test suite, so this use of a plain array is not supported. If you switch it to `Ember.A([a1, a2])`, the test passes.